lstchain/reco/r0_to_dl1.py

```diff
@@ -257,6 +261,45 @@ def get_dl1(
                 dl1_container=dl1_container,
             )
 
+            if 'lh_fit_config' in config.keys():
```

Comment:
Break this out into a function, please don't add more to the spaghetti here.

Reply:
Separated the likelihood reconstruction from get_dl1()
Now the call to get_dl1_lh_fit is in a new function apply_lh_fit itself used in r0_to_dl1 after the hillas parameter extraction.